Makefile

```diff
@@ -77,6 +78,17 @@ test:
 	sed -i '1d' coverage.txt
 	sed -i '1s/^/mode: atomic\n/' coverage.txt
 
+integration-test:
+	@echo "Building operator integration tests"
+	@cd test/integration_test && go test -tags integrationtest -v -c -o operator.test
+
+integration-test-container:
+	@echo "Building container: docker build --tag $(STORAGE_OPERATOR_TEST_IMG) -f Dockerfile ."
+	@cd test/integration_test && sudo docker build --tag $(STORAGE_OPERATOR_TEST_IMG) -f Dockerfile .
```

Comment:
There is no Dockerfile in test/integration_test. This step will fail.
Please add the Dockerfile.

Reply:
Done